frontend/src/components/CustomForm/CustomForm.jsx

```diff
@@ -11,6 +12,16 @@ import {
   resetButton
 } from './CustomForm.module.scss';
 
+const onReset = form => {
+  const formFields = form.getRegisteredFields();
+  if (!isEmpty(formFields)) {
```

Comment:
Instead of doing this iteration, can we try using form.resetFieldState?
https://final-form.org/docs/final-form/types/FormApi

I think it should be something like:

```suggestion
  const onReset = form => {
      form.reset();
      form.resetFieldState();
  }
```

Reply:
I got confused with this comment, you were right that the function is used to reset field by field, Nice job!